buildtools/cli.go

```diff
@@ -717,7 +717,7 @@ func dockerCmd(c *cli.Context) error {
 	default:
 		err = dockerNativeCmd(c)
 	}
-	if err == nil {
+	if err == nil && cmd != "scan" {
```

Comment:
How come `cmd == scan` in this case when line 716 returns from this function for this case?